ui/lib/kv/addon/components/kv-paths-card.hbs

```diff
@@ -0,0 +1,82 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
```

Comment:
Originally this lived in `ui/lib/kv/addon/components/page/secret/paths.hbs` moved everything but the header to this component so it could render in the overview and the paths route

### condensed version
<img width="1052" alt="Screenshot 2024-08-16 at 10 24 17 AM" src="https://github.com/user-attachments/assets/c1841198-0e28-4634-93da-84be2304b771">


### uncondensed (page) version
<img width="1061" alt="Screenshot 2024-08-16 at 10 24 57 AM" src="https://github.com/user-attachments/assets/76971e35-05aa-4077-91c4-53d49d094bc7">